docs/faq.md

```diff
@@ -44,15 +44,11 @@ XMTP Labs is researching various consensus protocols that would allow the networ
 
 Yes, you will be able to run a node.
 
-XMTP Labs is working toward a phased decentralization of the network. To learn more see, [XMTP: The journey to decentralization](/blog/journey-to-decentralization).
+XMTP Labs is working toward a phased decentralization of the network. To learn more, see [XMTP: The journey to decentralization](/blog/journey-to-decentralization).
 
-### What is the relationship between Waku and XMTP?
+### How do XMTP clients communicate with XMTP nodes?
```

Comment:
At this point the nodes still do communicate using waku. This may change down the line, but as of today it is still use the waku relay and store protocols.